pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -55,6 +55,8 @@ type Manager interface {
 	// certificate manager, as well as the associated certificate and key data
 	// in PEM format.
 	Current() *tls.Certificate
+	// RotateCh returns the certificate rotate channel.
+	RotateCh() <-chan bool
```

Comment:
a channel here only allows for a single consumer, which doesn't seem right. do you want a sync.Cond instead?

Reply:
One consumer channel not wright? I donot think so. Do you think sync.Cond is better than channel here?

Reply:
@liggitt OK. I think about using sync.Cond here. Maybe better